.github/actions/java-test/action.yaml

```diff
@@ -68,7 +68,7 @@ runs:
       env:
         COMET_PARQUET_SCAN_IMPL: ${{ inputs.scan_impl }}
       run: |
-        MAVEN_OPTS="-XX:+UnlockDiagnosticVMOptions -XX:+ShowMessageBoxOnError -XX:+HeapDumpOnOutOfMemoryError -XX:ErrorFile=./hs_err_pid%p.log" SPARK_HOME=`pwd` ./mvnw -B clean install ${{ inputs.maven_opts }}
+        MAVEN_OPTS="-Xmx4G -Xms2G -XX:+UnlockDiagnosticVMOptions -XX:+ShowMessageBoxOnError -XX:+HeapDumpOnOutOfMemoryError -XX:ErrorFile=./hs_err_pid%p.log" SPARK_HOME=`pwd` ./mvnw -B clean install ${{ inputs.maven_opts }}
```

Comment:
This memory change did not help, but also did no harm

Reply:
Didn't help, do you mean the test fails on OOM?

Reply:
The Comet test suites fail with OOM when running on macOS. I tried this change to specify more memory, but it did not make any difference. The macOS workflow is commented out in this PR, and I filed a follow up issue https://github.com/apache/datafusion-comet/issues/1949